dot/rpc/modules/interfaces.go

```diff
@@ -3,7 +3,9 @@
 
 package modules
 
-import "encoding/json"
```

Comment:
is this needed diff?

Reply:
No just my dumb editor thinking it should do this 😢  Reverted it!